src/browser/BrowserService.cpp

```diff
@@ -1009,13 +1022,21 @@ QList<Entry*> BrowserService::searchEntries(const QString& siteUrl,
     QList<Entry*> entries;
     do {
         for (const auto& db : databases) {
-            entries << searchEntries(db, siteUrl, formUrl, passkey);
+            entries << searchEntries(db, siteUrl, formUrl, keys, passkey);
         }
     } while (entries.isEmpty() && removeFirstDomain(hostname));
 
     return entries;
 }
 
+QString BrowserService::decodeCustomDataRestrictKey(const QString& key)
+{
+    if (key.isEmpty())
```

Comment:
Let's keep this consistent with other code:
```
if (key.isEmpty()) {
    return tr("Disable");
} else {
    return key;
}
```

Or just:
`return key.isEmpty() ? tr("Disable") : key;`